static/js/gallery.js

```diff
@@ -1,7 +1,18 @@
+const baseUrl = window.location.origin;
+
 const removeTrailingSlash = (string) => {
   return string.endsWith("/") ? string.slice(0, -1) : string;
 };
 
+const composeAnalyticsLink = (userId) => {
+  const galleryLink = document.getElementById("gallery-analytics-link");
+  galleryLink.innerHTML = `
+  <a href="${baseUrl}/analytics/${userId}">
+      <button class="button header-button">Analytics</button>
```

Comment:
nit: "User Analytics" (Similar rationale to the above suggestion)

Reply:
Done